### ISSUE TITLE
[Suggestion] Expliciter l'effet des niveaux des pièces d'équipement et de leur "slot"

### ISSUE THREAD

**Issue by Simon-Lessard** [JulienNigon/SpellcasterUniversity-Bug-Report#891]
Le tooltip du niveau des pièces d'équipement devrait donner l'éeffet en cours et au prochain niveau.

Aussi, ce n'est pas clair si les étudiants peuvent ou non avoir plusieurs pièces d'équipement pour un même "slot". Par exemple, si un étudiant peut avoir 2+ staffs. Selon mes tests ce n'est pas le cas. Si c'est par design, il serait bien de classer les pièces par slots dans l'écran d'équipement et de potentiellement en désactiver certaines pour qu'elles ne soient pas distribuées au étudiants.

**Comment by MlleMau** [comment]
Bonjour Simon-Lessard,

En vérité, lorsque l'on améliore un équipement, l'effet n'augmente pas, mais ce sont les conditions pour l'avoir qui diminuent. Par exemple, il faut 3 en Magie Sacrée pour avoir l'Auréole. Une Auréole niveau 2 ne nécessite qu'un niveau 2 en Magie Sacrée. 

Honnêtement, je n'ai jamais fait attention de si un élève pouvait avoir deux bâtons différents ! Tiens, il faudra que je fasse le test. Certes graphiquement, il est possible de voir si un élève en est équipé, mais tu peux aussi voir les équipements d'un élève en cliquant dessus. Au-dessus de ses traits, tu as une barre avec tous les bonus et les malus de l'élève, et les équipements apparaissent. Tu peux ainsi voir s'il a 2 bâtons différents ou pas, si tu te sens de faire le test avant moi !

**Comment by Simon-Lessard** [comment]
Bonjour,

J'ai fait le test avec la barre des buffs/debuffs dans la page statut des étudiants, pas seulement leur version graphique, mais mes tests n'étaient pas parfaits non plus donc mieux de contre-vérifier.

Je sais plus ou moins pour les conditions plutôt que l'effet, mais le truc est que les niveaux fonctionnent sur toutes les pièces alors que certaines sont données dès le niveau 1 (e.g., Arcanic Amulet). Je pense que les niveaus sont simplement inutiles pour cette dernière mais difficile d'en être sûr à 100% sans regarder dans le code.

![niveau-equipement](https://user-images.githubusercontent.com/23580881/75996233-ddd9d980-5efd-11ea-8218-68312b5f1783.png)

**Comment by MlleMau** [comment]
Pour les équipements ne demandant qu'un niveau de magie, il est possible, en l'améliorant, qu'un élève puisse l'avoir en ayant un niveau 0. En tout cas, je l'ai déjà constaté durant une partie où j'avais notamment le grimoire qui baissait tous les niveaux de 1 pour s'équiper d'un objet. Ouaip, c'est une piste à approfondir !

**Comment by Neelan22** [comment]
Effectivement, il n'est pas utile d'avoir plusieurs équipements de niveau 1 (comme l'amulette arcanique). Dans le cas ou un élève peut porter 2 bâtons, il ne portera que celui de plus haut niveau. Les effets de l'autre bâton ne seront pas ajoutés en buff. 
Nous sommes conscient que le mécanisme actuel n'est pas optimal et nous réfléchissons à comment l'améliorer sans trop frustrer les joueurs.

**Comment by Simon-Lessard** [comment]
Ah, merci pour la précision pour le bâton le plus haut niveau, ça me permettra d'ajsuter mes choix pour ne pas avoir uniquement le bâton arcanique.
